pykeops/torch/nn/ivf.py

```diff
@@ -0,0 +1,69 @@
+from pykeops.common.ivf import GenericIVF
+import torch
+
```

Comment:
The comments for the NumPy implementation also apply to this file.

Reply:
done